### PR TITLE
Fix gluster system and cluster dashboard with no volumes

### PR DESCRIPTION
**Pull request by anmolbabu**
tendrl-bug-id: Tendrl/performance-monitoring#181
Signed-off-by: anmolbabu <anmolbudugutta@gmail.com>

### CODE REVIEW

File: tendrl/performance_monitoring/sds/glusterfs/glusterfs_plugin.py [hunk 1]
```diff
@@ -292,6 +294,8 @@ def get_cluster_summary(self, cluster_id, cluster_name):
             ).value
         except EtcdKeyNotFound:
             pass
+        if not connection_active:
+            connection_active = 0
```
Comment (shtripat) [inline tendrl/performance_monitoring/sds/glusterfs/glusterfs_plugin.py]: I feel below we can say something like `ret_val['connection_active'] = connection_active or 0`.
You can try once and update accordingly in all places.

Comment (anmolbabu) [inline tendrl/performance_monitoring/sds/glusterfs/glusterfs_plugin.py]: Done

**Comment by anmolbabu** [comment]
Verified

**Comment by anmolbabu** [comment]
@shtripat @nthomas-redhat @r0h4n Please review

**Pull request merged**